gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
```

Comment:
Should be `ALL` distribution right?

Reply:
This wrapper is used for samples project. Plugin project (that is included into samples project as composite build) uses `ALL` distribution.